styleguide/derek/examples/layouts/additional-resources.ejs

```diff
@@ -1,20 +1,17 @@
 <!-- Additional Resources: 33/34/33 -->
 <div class="container half-padding-full">
-  <h4>Additional Resources</h4>
   <div class="row row-eq-height">
     <div class="col-sm-4 panel-flex">
       <%- partial('../../_partials/cards/additional-resource.ejs') %>
     </div>
     <div class="col-sm-4 panel-flex">
       <%- partial('../../_partials/cards/additional-resource.ejs', {
-        type: 'video',
-        desc: 'This is short'
+        type: 'video'
       }) %>
     </div>
     <div class="col-sm-4 panel-flex">
       <%- partial('../../_partials/cards/additional-resource.ejs', {
-        type: 'rss',
-        desc: 'This is extra long. This is extra long. This is extra long. This is extra long. This is extra long. This is extra long. This is extra long. This is extra long. This is extra long. This is extra long.'
```

Comment:
Im adding shortened lengths but not super long ones because the ipsum ive put in is the max allowable characters so I dont want to show anything longer than that.